java/vector/src/main/codegen/templates/BaseWriter.java

```diff
@@ -117,7 +117,7 @@ public interface StructOrListWriter {
     void start();
     void end();
     StructOrListWriter struct(String name);
-    StructOrListWriter listoftstruct(String name);
+    StructOrListWriter listofstruct(String name);
```

Comment:
This is a public method. I don't think we can just replace it, we should deprecate the old one and add the new one.

Reply:
would you prefer `listOfStruct` or `listofstruct`

Reply:
`listOfStruct` would be better if we're already doing a rename.